docs/en/observability/synthetics-scaling-and-architecting.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[synthetics-scaling-and-architecting]]
+= Scaling and Architecting a Synthetics Deployment
+
+While for simple use cases little to no planning needs to be taken in the implementation of *{synthetics-app}*, 
+for complex use cases and at a high scale this section describes advanced considerations for any *{synthetics-app}*
+implementation
+
+[discrete]
+[[synthetics-no-ccs-ccr]]
+= *{synthetics-app}* does not work with CCS/CCR
+
+In complex environments a common pattern may be to have multiple task-specific Elastic stack deployments with one centralized
+overview cluster using CCS or CCR to centralize Kibana dashboards and apps. *{synthetics-app}* by design does not support this
+pattern. Configure your Synthetic monitors directly on the Kibana instance you want to view / manage them on instead.
+
+This limitation is a requirement for the rich user experience *{synthetics-app}* provides.
+Unlike the majority of Kibana apps *{synthetics-app}* relies heavily on state stored in Kibana shared objects in order
+to provide a rich user experience. Since Kibana Saved Objects cannot be shared via CCS/CCR the *{synthetics-app}* will not show
+any user data even if CCS/CCR is configured.
```

Comment:
I would restructure/reword a bit. Question: are "Kibana shared objects" and "Kibana Saved Objects" the same thing? 

```suggestion
In complex environments it's common to have multiple task-specific {stack}
deployments with one  centralized overview cluster using CCS or CCR to centralize {kib}
dashboards and apps. *Do not use this pattern with the {synthetics-app}*.
Instead, configure your synthetic monitors directly on the {kib} instance where you
want to view and manage them.

The {synthetics-app} does _not_ work with CCS/CCR because it would limit the rich
user experience that the {synthetics-app} provides.
Unlike the majority of {kib} apps, the {synthetics-app} relies heavily on state stored
in {kib} shared objects in order to provide a rich user experience.
Because {kib} saved objects cannot be shared via CCS/CCR, the {synthetics-app}
will not show any user data even if CCS/CCR is configured.
```